src/main/java/org/ow2/proactive/catalog/dto/BucketGrantMetadata.java

```diff
@@ -33,10 +33,12 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.Setter;
 
 
 @Data
 @Getter
+@Setter
```

Comment:
The setter of all fields are used ? If not please annotate only the required fields with @Setter

Reply:
I'll remove it actually. I don't need it anymore.